File: src/main/java/ru/zdoher/japs/repositories/WordRepositories.java
package ru.zdoher.japs.repositories;

import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;
import ru.zdoher.japs.domain.Word;
import ru.zdoher.japs.repositories.aggregation.RepositoryAggregation;

import java.util.List;

@Repository
public interface WordRepositories
        extends CrudRepository<Word, String>, RepositoryAggregation<Word> {

    List<Word> findAll();

}

File: src/main/java/ru/zdoher/japs/repositories/WordRepositories.kt
package ru.zdoher.japs.repositories

import org.springframework.data.mongodb.repository.ReactiveMongoRepository
import org.springframework.stereotype.Repository
import ru.zdoher.japs.domain.Word
import ru.zdoher.japs.repositories.aggregation.RepositoryAggregation

@Repository
interface WordRepositories : ReactiveMongoRepository<Word, String>, RepositoryAggregation<Word>